lib/Models/Internationalization.ts

```diff
@@ -148,16 +150,14 @@ class Internationalization {
 
           // cache user language on
           caches: ["localStorage"],
-          excludeCacheFor: ["cimode"], // languages to not persist (cookie, localStorage)
+          excludeCacheFor: ["cimode"] // languages to not persist (cookie, localStorage)
 
           // optional expire and domain for set cookie
           // cookieMinutes: 10,
           // cookieDomain: "myDomain",
-
-          // only detect languages that are in the whitelist
-          checkWhitelist: true
         },
```

Comment:
I assume `checkWhitelist: true` removed becasue this is the default anyway?

Reply:
Yeah commit message on i18next detection don't even mention this change they just say respect values set in main i18next config. And as I figured out they are now doing this by default

Reply:
also in i18next-browser-languageDetector v3.1.1 `checkWhitelist` was set to true by default